adminVerify.php

```diff
@@ -0,0 +1,18 @@
+<?php
+require_once('cmsController.php');
+$db = callDatabase();
+
+
+session_start();
+
+require('adminFunctions.php');
+
+$enteredUser=$_POST['username'];
+
+$enteredPassword=$_POST['password'];
+
+$_SESSION['loggedIn']=logIn($enteredUser,$enteredPassword,$db);
```

Comment:
just pass `$_POST['username']` here instead of using an unnecessary variable

Reply:
same for password....obvs

Reply:
Resolved